lib/situation_room/check/check_make.ex

```diff
@@ -0,0 +1,25 @@
+defmodule SituationRoom.Check do
```

Comment:
Need a better name here.  `Check` is ambiguous and doesn't really indicate what this module is or does.

Also, the file name doesnt match the module namespace path either, and these should match to be idiomatic.